discovery-provider/src/solana/audius_data_transaction_handlers.py

```diff
@@ -243,6 +243,113 @@ class ManageEntityData(TypedDict):
     user_id_seed_bump: Any
 
 
+def create_track_model(instruction_data, slot, txhash, created_at, updated_at):
+    new_track = Track(
+        slot=slot,
+        txhash=txhash,
+        track_id=instruction_data["id"],
+        owner_id=instruction_data["user_id_seed_bump"].user_id,
+        metadata_multihash=instruction_data.get("metadata"),
+        is_current=True,
+        is_delete=False,
+        created_at=created_at,
+        updated_at=updated_at,
+    )
+    return new_track
+
+
+def create_playlist_model(instruction_data, slot, txhash, created_at, updated_at):
+    new_playlist = Playlist(
+        slot=slot,
+        txhash=txhash,
+        playlist_id=instruction_data["id"],
+        playlist_owner_id=instruction_data["user_id_seed_bump"].user_id,
+        metadata_multihash=instruction_data.get("metadata"),
+        is_current=True,
+        is_delete=False,
+        created_at=created_at,
+        updated_at=updated_at,
+    )
+    return new_playlist
+
+
+def create_entity(entity_type, instruction_data, transaction):
+    ENTITY_TYPE_TO_CREATE_MODEL_HANDLER = {
```

Comment:
this is fine but can we add some explanation as to what is happening here? also there is definitely some missing stuff i think

Reply:
might be good to consolidate these and ENTITY_TYPE_TO_NAME in anchor_program_indexer. actually we could probably get away with just that and say if entity type is track call create_track_model etc since there's just a few types.